src/resources/armies/borgo.json

```diff
@@ -60,10 +65,151 @@
       ],
       "modifiers": [
         {
-          "type": "initiative",
-          "value": -128
+          "type": "net"
         }
       ]
+    }, {
+      "id": "mutant",
+      "initiative": 2,
+      "health": 2,
+      "count": 6,
+      "attacks": [
+        {
+          "type": "melee",
+          "damage": 1
```

Comment:
1 is the default.

Reply:
Removed damage tag everywhere where it was 1